src/lib.rs

```diff
@@ -91,37 +109,30 @@
 //!
 //! > TODO: although the WalkDir error can be auto-converted to std::io::Error, it
 //! > does not preserve the pretty output. See
-//! > [this ticket](https://github.com/BurntSushi/walkdir/pull/91)
+//! > [this ticket](https://github.com/BurntSushi/walkdir/pull/92)
 //!
 //! ### Examples
 //! ```rust
 //! # extern crate ergo_fs;
-//! use ergo_fs::{WalkDir, PathType};
+//! use ergo_fs::*;
 //!
 //! # fn try_main() -> ::std::io::Result<()> {
-//! // note: this will error
```

Comment:
huh, I must have forgot to push this stuff. Oh well, it can go in this review too.